internal/indexmanagement/reconcile.go

```diff
@@ -32,6 +33,7 @@ import (
 const (
 	indexManagementConfigmap = "indexmanagement-scripts"
 	defaultShardSize         = int32(40)
+	workingDir = "/tmp/scripts"
```

Comment:
surprised this wasn't auto-formatted to be consistent with the above lines

Reply:
fixed